WORKSPACE

```diff
@@ -83,13 +83,6 @@ gemfile_lock_deps(
     ],
 )
 
-load("@rules_rust//rust:repositories.bzl", "rust_repositories")
-
-rust_repositories(
-    edition = "2018",
-    version = "1.51.0",
-)
```

Comment:
This is a small request, feel free to decline: what do you think about deleting the now-dead code in a separate PR?

If in the future we decide we want to change our mind for whatever reason, it might be nice to be able to revert _just_ the bazel/rust deletions

Reply:
concretely, maybe we could defer the deletions in `WORKSPACE`, `experimental/rubyfmt/`, and `third_party/` to a future PR, and only change the core of sorbet in this PR?

Reply:
Sounds good, I'll put it into a separate PR 👍